PowerShell/JumpCloud Commands Gallery/Mac Commands/Application Installs/Mac - Install Microsoft Office.md

```diff
@@ -73,34 +76,19 @@ fi
 
 # Test to ensure App is not already installed
 # We're hard coding the apps as they're not included in the pkg
-ExistingSearch=$(find "/Applications/" -name "Microsoft\ Word.app" -depth 1)
-
-if [ -n "$ExistingSearch" ]; then
-
-    echo "Microsoft Word already present in /Applications folder"
-    rm -r /tmp/$TempFolder
-    echo "Deleted /tmp/$TempFolder"
-    exit 1
-
-else
-
-    echo "Microsoft Word not present in /Applications folder"
-fi
-
-ExistingSearch=$(find "/Applications/" -name "Microsoft\ Outlook.app" -depth 1)
-
-if [ -n "$ExistingSearch" ]; then
-
-    echo "Microsoft Outlook already present in /Applications folder"
-    rm -r /tmp/$TempFolder
-    echo "Deleted /tmp/$TempFolder"
-    exit 1
-
-else
-
-    echo "Microsoft Outlook not present in /Applications folder"
-fi
+list=("Microsoft Excel" "Microsoft OneNote" "Microsoft Outlook" "Microsoft PowerPoint" "Microsoft Word")
+for n in "${list[@]}"; do
+    app="/Applications/$n.app"
+    if [[ -d "$app" ]]; then
+        echo "$n already present in /Applications folder"
+        rm -r /tmp/$TempFolder
```

Comment:
Would we want `rm -rf` here, just to suppress any messages?

Reply:
I think that's fine, it works without but that always assumes that the tmp/$TempFolder exists.

Reply:
The only difference would be that rm -r would return:
rm: /tmp/theFOlderName : No such file or directory
which may be useful for diagnosis. Either really works in this case I believe.